hera/nfs/shared.py

```diff
@@ -0,0 +1,8 @@
+from hera.shared import global_config
+import urllib3
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
+global_config.host = "https://localhost:8443/argo/"
+global_config.token = "unused"
+global_config.verify_ssl = False
+namespace = 'argo'
```

Comment:
Is this supposed to be global?

```suggestion
global_config.namespace = 'argo'
```

Reply:
Done (although there's only one workflow here, and only one template, it's still nice to show)